releases/release-1.30/release-notes/release-notes-draft.json

```diff
@@ -2158,5 +2374,384 @@
+    "markdown": "Fixed deprecated version for `pod_scheduling_duration_seconds` that caused the metric to be hidden by default in `1.29`. ([#123038](https://github.com/kubernetes/kubernetes/pull/123038), [@alculquicondor](https://github.com/alculquicondor))",
+    "author": "alculquicondor",
+    "author_url": "https://github.com/alculquicondor",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/123038",
+    "pr_number": 123038,
+    "areas": [
+      "stable-metrics"
+    ],
+    "kinds": [
+      "bug",
+      "regression"
+    ],
+    "sigs": [
+      "scheduling",
+      "instrumentation"
+    ],
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "123052": {
+    "commit": "77775d45469a3da6a7ce5e6e2221cff17c2d27fe",
+    "text": "Fixed cleanup of Pod volume mounts when a file was used as a subpath.",
+    "markdown": "Fixed cleanup of Pod volume mounts when a file was used as a subpath. ([#123052](https://github.com/kubernetes/kubernetes/pull/123052), [@jsafrane](https://github.com/jsafrane)) [SIG Node]",
+    "author": "jsafrane",
+    "author_url": "https://github.com/jsafrane",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/123052",
+    "pr_number": 123052,
+    "areas": [
+      "kubelet"
+    ],
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "node"
+    ]
+  },
+  "123055": {
+    "commit": "440f11d83dd939afb53d47a64fb1d6bf837cc50b",
+    "text": "Fixed an error when trying to expand a volume that does not require node expansion",
+    "markdown": "Fixed an error when trying to expand a volume that does not require node expansion ([#123055](https://github.com/kubernetes/kubernetes/pull/123055), [@gnufied](https://github.com/gnufied))",
+    "author": "gnufied",
+    "author_url": "https://github.com/gnufied",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/123055",
+    "pr_number": 123055,
+    "areas": [
+      "kubelet"
+    ],
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "storage",
+      "node"
+    ],
+    "duplicate": true
+  },
+  "123069": {
+    "commit": "003187fd5a0803eb25f984e07a5c7c47a5621c1d",
+    "text": "Build etcd image `v3.5.12`",
+    "markdown": "Build etcd image `v3.5.12` ([#123069](https://github.com/kubernetes/kubernetes/pull/123069), [@bzsuni](https://github.com/bzsuni))",
+    "author": "bzsuni",
+    "author_url": "https://github.com/bzsuni",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/123069",
+    "pr_number": 123069,
+    "areas": [
+      "release-eng"
+    ],
+    "kinds": [
+      "cleanup"
+    ],
+    "sigs": [
+      "api-machinery",
+      "etcd"
+    ],
+    "duplicate": true
+  },
+  "123077": {
+    "commit": "173cf189209a2ad21cf251164275cd3cc5366b74",
+    "text": "kubeadm: fix a bug where the --rootfs global flag does not work with \"kubeadm upgrade node\" for control plane nodes.",
```

Comment:
Do kubeadm release notes not get grouped with the other release notes? Just curious- I'm not seeing this one in the yaml files.

Reply:
@sanchita-07 Do you have any insight into this? Other kubeadm release notes have yaml files but this one doesn't.

Reply:
The map files are created after the modification of a release-note during the interactive mode. It might be the case:
1. The release-note is not modified during the interactive mode.
2. The release-note may be modified in the previous version.

Reply:
Thanks for the explanation! It just caught my eye because of the `fix`, but then I didn't see the release note in the maps directory. I don't see this note already in the `sig-release`, so I think the next run will just pull it in and it can be fixed there.